jsxc.lib.js

```diff
@@ -710,12 +716,7 @@ var jsxc;
       translate: function(text) {
          return text.replace(/%%([a-zA-Z0-9_-}{ .!,?/'@]+)%%/g, function(s, key) {
             var k = key.replace(/ /gi, '_').replace(/[.!,?/'@]/g, '');
-
-            if (!jsxc.l[k]) {
-               jsxc.warn('No translation for: ' + k);
```

Comment:
do we need the warning? We still need the function for translating the templates.